content/administration/maintain/azure_oauth.rst

```diff
@@ -0,0 +1,170 @@
+
+Click on :guilabel:`Add` when these two values are entered. A :guilabel:`Client Secret Value` and
+:guilabel:`Secret ID` will be created. It is important to copy the :guilabel:`Value` or
+:guilabel:`Client Secret Value` into a notepad as it will become encrypted after leaving this page.
+The :guilabel:`Secret ID` is not needed.
+
+.. image:: azure_oauth/secretvalue.png
+   :align: center
+   :alt: Client Secret Value or value in the app's credentials.
+
+API Permissions
+---------------
+
+The :guilabel:`API Permissions` should be set next. The :guilabel:`API permissions` link is located
+in the :guilabel:`Left Menu`. Click on the :guilabel:`(+) Add a Permission` button and select
+:guilabel:`Microsoft Graph` under :guilabel:`Commonly Used Microsoft APIs`. After doing so, select
+the :guilabel:`Delegated Permissions` option.
+
+:guilabel:`Search` and :guilabel:`Add permissions` for the following :guilabel:`Deregulated
+permissions`:
+
+- :guilabel:`SMTP.Send`
+- :guilabel:`IMAP.AccessAsUser.All`
+
+The :guilabel:`User.Read` permission will be added by default.
+
+.. image:: azure_oauth/permissions.png
+   :align: center
+   :alt: API permissions added for the app.
+
+Assign Users and Groups
+=======================
+
+Navigate back to the :guilabel:`Overview` of the :guilabel:`Application` in the top of the menu on
+the left-hand side. It's time to add users onto this application. Under the :guilabel:`Essentials`
+overview table click on the link :guilabel:`Managed Application in Local Directory`, or the last
+option on the bottom right-hand side of the table.
+
+.. image:: azure_oauth/managed-application.png
+   :align: center
+   :alt: Link to the created application and to add users/groups.
+
+In the :guilabel:`Left Menu` select :guilabel:`Users and Groups`. Next select the :guilabel:`(+)
+Add User/Group`. Depending on the account, either a :guilabel:`Group` and :guilabel:`User` can be
+added, or :guilabel:`Users` alone. Personal accounts will only allow for users to be added.
+
+Add the users or group of users that will be sending from the :guilabel:`Microsoft account` in
+:guilabel:`Odoo` by clicking on :guilabel:`None Selected` under users or groups. :guilabel:`Add`
+the users/groups, click :guilabel:`Select` and then :guilabel:`Assign` them to the application.
+
+This completes the setup on the :guilabel:`Microsoft Azure Portal` side.
+
+Setup in Odoo
+=============
+
+Enter Microsoft Outlook Credentials
+-----------------------------------
+
+First, open :guilabel:`Odoo` and navigate to the :guilabel:`Apps module`. Then, remove the Apps
+filter from the :guilabel:`search bar` and type in `Outlook`. Install the module called
+:guilabel:`Microsoft Outlook`.
+
+Next, navigate to :guilabel:`Settings ‣ General Settings`, and under the :guilabel:`Discuss`
+section, ensure that the checkbox for :guilabel:`Custom Email Servers` is checked. This populates a
+new option for :guilabel:`Outlook Credentials`.
```

Comment:
```suggestion
Next, navigate to :guilabel:`Settings --> General Settings`, and under the :guilabel:`Discuss`
section, ensure that the checkbox for :guilabel:`External Email Servers` is checked. This populates a
new option for :guilabel:`Outlook Credentials`.
```

Nitpick: this PR is targeting the `14.0` branch, and in Runbot 14.0, this field is called "External Email Servers" instead of "Custom Email Servers. I would make note of this so you can open a new PR to correct it for 15.0 user docs, after this PR is merged and forward-ported.

Reply:
Note: [Task #3048351](https://www.odoo.com/web#id=3048351&cids=3&menu_id=4720&action=333&active_id=6050&model=project.task&view_type=form)